cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/SlowBrokerFinder.java

```diff
@@ -174,6 +184,7 @@ public SlowBrokerFinder() {
     }
 
     Set<BrokerEntity> detectMetricAnomalies = getMetricAnomalies(historicalLogFlushTimeMetricValues, currentLogFlushTimeMetricValues);
+    detectMetricAnomalies.retainAll(detectMetricAnomaliesFromThreshold(currentLogFlushTimeMetricValues, _logFlushTimeThresholdMs));
```

Comment:
Can we avoid passing private variable `_logFlushTimeThresholdMs` to non-static function `detectMetricAnomaliesFromThreshold` and update the name of the function to indicate its custom role regarding log flush time?

Actually it is a little confusing that we have another function in this class called `getMetricAnomalies`, which is also used for detecting metric anomalies -- can we merge/rename them to make the logic more clear?

Reply:
I rename the method to make it more specific.
I was thinking about merge this part of logic into `getMetricAnomalies()`. The issue is the logic in `getMetricAnomalies()` is generic and this extra check is specific to log flush time metrics.